js/render/draw_symbol.js

```diff
@@ -197,13 +203,15 @@ function drawSymbol(painter, layer, posMatrix, tile, bucket, bufferGroups, isTex
         gl.uniform1f(program.u_bearing, tr.bearing / 360 * 2 * Math.PI);
         gl.uniform1f(program.u_aspect_ratio, tr.width / tr.height);
 
+        painter.setDepthSublayer(1);
```

Comment:
I think that could be changed in -native. My guess is either:
- depthMask = false wasn't used when it was first written, in which case it would be needed
- or, it's never been needed but it's never caused problems